source/blog/2017-08-16-ovirt-ansible-roles-how-to-use.html.md

```diff
@@ -0,0 +1,153 @@
+
+## Ansible Galaxy
+
+[Ansible Galaxy](https://galaxy.ansible.com/) stores many different Ansible roles, making your life simpler. oVirt Ansible roles are also stored in Galaxy. To install those roles from Ansible Galaxy, perform the following steps:
+
+To install roles on your local machine, run the following command:
+
+```bash
+$ ansible-galaxy install ovirt.ovirt-ansible-roles
+```
+
+This will install your roles into directory `/etc/ansible/roles/ovirt.ovirt-ansible-roles/`. By default, Ansible only searches for roles in `/etc/ansible/roles/` directory and your current working directory. To change the directories where Ansible looks for roles, modify the `roles_path` option of `[defaults]` section in `ansible.cfg` configuration file. The default location of this file is in `/etc/ansible/ansible.cfg`.
+
+```bash
+$ sed -i 's|#roles_path    = /etc/ansible/roles|roles_path = /etc/ansible/roles:/etc/ansible/roles/ovirt.ovirt-ansible-roles/roles|'  /etc/ansible/ansible.cfg
+```
+
+For more information on changing the directories where Ansible searches for roles, see the Ansible documentation pages.
+
+Copy one of the examples from the directory `/etc/ansible/roles/ovirt.ovirt-ansible-roles/examples/` into your working directory, then modify the needed variables and run the playbook.
+
+## oVirt Ansible Roles RPM
+
+In the latest oVirt repositories you can find the `ovirt-ansible-roles` package. To make the oVirt repositories available, install `ovirt-release` RPM:
+
+```bash
+$ yum install -y http://resources.ovirt.org/pub/yum-repo/ovirt-release41.rpm
+```
+
+oVirt release repository now provides `ovirt-ansible-roles` package:
+
+```bash
+$ yum install -y ovirt-ansible-roles
+```
+
+The structure of the `ovirt-ansible-roles` package is as follows:
+
+ - __/usr/share/ansible/roles__ - stores the roles
+ - __/usr/share/doc/ovirt-ansible-roles/__ - stores the examples, a basic overview and the licence
+ - __/usr/share/doc/ansible/roles/{role_name}__ - stores the documentation specific to the role
+
+Copy one of the examples from the directory `/etc/ansible/roles/ovirt.ovirt-ansible-roles/examples/` into your working directory, then modify the needed variables and run the playbook.
+
+## Example of tweaking and running the playbook
+
+### Create example playbook
+
+The proccess of runnning the example from galaxy and from RPM is bit different so please follow relevant part.
+
+#### Ansible Galaxy
+
+The following steps assume that you followed the steps on how to install the roles from Galaxy, your roles are installed in `/etc/ansible/roles/ovirt.ovirt-ansible-roles/roles/`, and you have tweaked `ansible.cfg` to look for those roles.
+
+Run a role called `ovirt-image-template`, to download the qcow2 image and create a template from it.
```

Comment:
Please change:
"Run a role called `ovirt-image-template` to download the qcow2 image and create a template from it."
(remove comma after "template"